service_container.rst

```diff
@@ -700,6 +700,100 @@ For a full list of *all* possible services in the container, run:
 
     $ php bin/console debug:container
 
+Injecting a Closure as an Argument
+----------------------------------
+
+It is possible to inject a callable as an argument of a service.
+Let's add an argument to our ``MessageGenerator`` constructor::
+
+    // src/Service/MessageGenerator.php
+    namespace App\Service;
+
+    use Psr\Log\LoggerInterface;
+
+    class MessageGenerator
+    {
+        private $logger;
+        private $messageHash;
+
+        public function __construct(LoggerInterface $logger, callable $generateMessageHash)
+        {
+            $this->logger = $logger;
+            $this->messageHash = $generateMessageHash();
+        }
+        // ...
+    }
+
+Now, we would add a new invokable service to generate the message hash::
+
+    // src/Hash/MessageHashGenerator.php
+    namespace App\Hash;
+
+    class MessageHashGenerator
+    {
+        public function __invoke(): string
+        {
+            // Compute and return a message hash
+        }
+    }
+
+Now, our configuration would look like this:
```

Comment:
We can maybe avoid repeat `Now, ...` which is use just above

Reply:
You're right, updated!